Formula/homebrew-dns-scout.rb

```diff
@@ -0,0 +1,19 @@
+class HomebrewDnsScout < Formula
+  desc "DNS Scout for Linux/MacOS pulls and displays DNS records"
+  homepage "https://github.com/careyjames/DNS-Scout"
+  url "https://github.com/careyjames/DNS-Scout/releases/download/v5.8/DNS-Scout-v5.8.tar.gz"
+  sha256 "e8d92f653069562699be58ca4fce421c645e7389aa7603fe786de046fbd3dc60"
+  license "MIT"
+  depends_on "go"
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    system "go", "build", "-o", "#{bin}/dns-scout"
+    chmod 0555, "#{bin}/dns-scout"
+  end
+
+  test do
+    system "#{bin}/dns-scout", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula